pandas/core/apply.py

```diff
@@ -308,11 +312,19 @@ def apply_standard(self):
                 # reached via numexpr; fall back to python implementation
                 pass
             else:
-                return self.obj._constructor_sliced(result, index=labels)
+                return_result = self.obj._constructor_sliced(result, index=labels)
```

Comment:
what are you trying to do here? we have a specific subclass for Series where this could live, but I am not clear what you are trying to catch